org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/ProjectManager.java

```diff
@@ -215,4 +215,12 @@ public ProjectDescription getProjectDescription() {
 	public IProjectConfig getProjectConfig() {
 		return projectConfig;
 	}
+	
+	public void delete() {
```

Comment:
I had another expectation from the method name `delete`.
Maybe `removeAllIssues` or `discardAllIssues`?

Reply:
Yes, I thought this more to be the hook to cleanup when the project is deleted from the workspace, but as long as it just discards issues, I can rename it.

Reply:
What do you think about `aboutToDeleteFromWorkspace()`?